checks/checker.go

```diff
@@ -94,5 +96,8 @@ func (c Checker) Check() (*Report, error) {
 // Interval is the interval where the command is invoked.
 // (Will be configurable in the future)
 func (c Checker) Interval() time.Duration {
```

Comment:
limit the range of return values.
It should be between 1 to 60 minutes.

If the configured value is out of range, 
truncate it to minimum or maximum value in the range
with warning logs.